wolfssl/wolfcrypt/rsa.h

```diff
@@ -352,28 +352,28 @@ WOLFSSL_API int  wc_RsaPSS_VerifyInline(byte* in, word32 inLen, byte** out,
 WOLFSSL_API int  wc_RsaPSS_VerifyInline_ex(byte* in, word32 inLen, byte** out,
                                            enum wc_HashType hash, int mgf,
                                            int saltLen, RsaKey* key);
-WOLFSSL_API int  wc_RsaPSS_Verify(byte* in, word32 inLen, byte* out,
+WOLFSSL_API int  wc_RsaPSS_Verify(const byte* in, word32 inLen, byte* out,
```

Comment:
I am not sure we can or should change public facing API's, even adding 'const`. I can guarentee this will break builds for some customers.

Reply:
It will break API compatibility. I'm fairly certain we cannot accept the changes where a public WOLFSSL_API is modified even to add const.